docs/authentication.md

```diff
@@ -4,33 +4,46 @@ sidebar_position: 2
 
 # Authentication
 
-The MobilePay API uses API keys to authenticate requests. They are used to get unlimited access to resources in your account.
+We are providing two different options for authentification. The first one is [OAuth access token](#get-an-oauth-access-token) which is used to get authenticated and scoped access to any MobilePay merchant account. Use them when your application accesses resources on behalf of account owners. I.e. you are an integrator. Second option is [API keys](#get-an-api-key). They are used to get unlimited access to resources in your merchant account. But firstly, we need to onboard app payments product.
 
-OAuth access tokens are used to get authenticated and scoped access to any MobilePay account. Use them when your application accesses resources on behalf of account owners. I.e. you are an integrator.
+## Onboarding App Payments application
+
+1. **Log-in on the developer portal**. Go to [developer portal](https://developer.mobilepay.dk/) and log in with your credentials.
+
+2. **Create an app in the developer portal**. Go to My Apps > Create new App to register a new application. You need to supply the x-ibm-client-id when calling APIs. You should always store the x-ibm-client-id in a secure location, and never reveal it publicly.
+
+3. **Subscribe the app to APIs.** Go to APIs and subscribe to APIs you need, e.g.:
```

Comment:
Subscribing is being done to the products. So you need to list products (Payments and Webhooks)